src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -185,13 +195,56 @@ void OrbitalWorkspace::loadWorkspaceConfig(const std::string &configPath) {
 
-void OrbitalWorkspace::saveWorkspaceConfig(const std::string &configPath) {
+	m_simulationConfigSaved = true;
 
+	updateCodeEditorTitles();
+
+
+	return true;
+}
+
+
+bool OrbitalWorkspace::saveWorkspaceConfig(const std::string &configPath, const std::string &data) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)